moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -135,18 +116,51 @@ servo:
                   due to the existence of a lever between the two frames."
   }
 
-  override_velocity_scaling_factor: {
-    type: double,
-    default_value: 0.0,
-    description: "Scaling factor when servo overrides the velocity (eg: near singularities)"
+############################ POSE TRACKING SETTINGS #############################
+
+  pose_tracking:
+    linear_tolerance: {
+        type: double,
+        default_value: 0.001,
+        description: "The allowable linear error when tracking a pose.",
+        validation: {
+          gt<>: 0.0
+        }
+    }
+
+    angular_tolerance: {
+        type: double,
+        default_value: 0.01,
+        description: "The allowable angular error when tracking a pose.",
+        validation: {
+          gt<>: 0.0
+        }
+    }
+
+############################## OUTGOING COMMAND SETTINGS #######################
+
+  status_topic: {
+    type: string,
+    read_only: true,
+    default_value: "~/status",
+    description: "The topic to which the status will be published"
   }
 
-  publish_period: {
-    type: double,
-    default_value: 0.034,
-    description: " 1 / (Nominal publish rate) [seconds]",
+  command_out_topic: {
+    type: string,
+    read_only: true,
```

Comment:
This one will definitely need to change when we enable changing planning groups. I'm OK if it's read only in this PR but we will very likely need to undo that later.

Reply:
Yeah, will keep it read-only for now.